### PR TITLE
Fix TokenMint usage estimate + update fee schedule for desired pricing

### PR DESCRIPTION
**Pull request by tinker-michaelj**
Addresses first check of #1766.

### CODE REVIEW

**Review by Neeharika-Sompalli — DISMISSED** [review]
LGTM ! Codecov complains and need to resolve conflicts

File: hapi-fees/src/main/java/com/hedera/services/usage/state/UsageAccumulator.java [hunk 1]
```diff
@@ -268,4 +294,46 @@ long getNumPayerKeys() {
...
+
+	private void setSbpr(long sbpr) {
+		this.sbpr = sbpr;
+	}
+
+	private void setVpt(long vpt) {
+		this.vpt = vpt;
+	}
+
+	private void setGas(long gas) {
```
Comment (anighanta) [inline hapi-fees/src/main/java/com/hedera/services/usage/state/UsageAccumulator.java:324]: nit : can remove this setter

**Review by anighanta — DISMISSED** [review]

**Review by abhishek-hedera — APPROVED** [review]

**Review by ljianghedera — DISMISSED** [review]
Looks great!

**Review by qnswirlds — DISMISSED** [review]
Need to resolve conflict.

**Pull request merged**